### PR TITLE
bump for season 4 start

### PR DESCRIPTION
**Pull request by ToppleTheNun**
### Description

Includes:
- bump patch compatibility for all specs that had 10.2.6 as their current spec
- add missing headshots for VotI
- marks 10.2.6 as no longer current
- marks current patch as 10.2.7
- updates e2e test log

### Testing

- Test report URL: `/report/akZLCTYbN2XpQFmg`
- Screenshot(s):
image

### CODE REVIEW

**Comment by emallson** [comment]
note that current patch is *not* 10.2.7. that releases in a couple weeks (i made the same mistake in SL)

**Comment by ToppleTheNun** [comment]
Oops you are right, I will fix this in the morning

File: e2e/report-selection.spec.ts
Comment (Seriousnes) [inline e2e/report-selection.spec.ts:1]: If it's just the `reportCode` and `fightCode` being updated throughout, maybe make add constants for it that only need to be changed in one spot.

E.g. use these and reference them in tests
```
const reportCode = 'bjqrZRnNdvKPXt13';
const fightCode = '34-Mythic+Fyrakk+the+Blazing+-+Kill+(8:47)';
```

File: src/analysis/retail/shaman/enhancement/CONFIG.tsx
Comment (Seriousnes) [inline src/analysis/retail/shaman/enhancement/CONFIG.tsx:1]: I'm handling this one in #6738

**Pull request merged**